src/components/StoreIssueBlock.js

```diff
@@ -25,18 +26,25 @@ class StoreIssueBlock extends Component {
   }
 
   onIssueClick() {
-    const { adminAddress, storeAddress, handleCertificateIssue } = this.props;
+    const {
+      adminAddress,
+      storeAddress,
+      accountBalance,
+      handleCertificateIssue
+    } = this.props;
     const { certificateHash } = this.state;
     if (isValidCertificateHash(certificateHash)) {
       handleCertificateIssue({
         storeAddress,
         fromAddress: adminAddress,
-        certificateHash
+        certificateHash,
+        accountBalance
       });
     } else {
       this.setState({
         certificateHashIsValid: isValidCertificateHash(certificateHash)
       });
+      toast.error("Merkle root is not valid.");
```

Comment:
Do you need both toast and red text to tell user the same message? Seemed like overkill

Reply:
Removed.